### PR TITLE
Switch to older associated domain format

### PR DESCRIPTION
**Pull request by whaught**
Fixes #https://github.com/google/exposure-notifications-verification-server/issues/1299

## Proposed Changes

* Use the older format for iOS Associated Domains
/hold to see if we can get confirmation from Apple on this (docs are confusing)

**Release Note**

```release-note
Switch to older Associated Domains format for iOS
```

### CODE REVIEW

**Review by mikehelmick — APPROVED** [review]

File: docs/link_handling.md [hunk 1]
```diff
@@ -48,15 +48,8 @@ Sample iOS universal link metadata
 	"applinks": {
 		"details": [
```
Comment (wescarr) [inline docs/link_handling.md:49]: According to the docs the following key must exist:
```json
"apps": []
```
From the docs
> The apps key in an apple-app-site-association file must be present and its value must be an empty array
https://developer.apple.com/library/archive/documentation/General/Conceptual/AppSearch/UniversalLinks.html

File: docs/link_handling.md [hunk 2]
```diff
@@ -48,15 +48,8 @@ Sample iOS universal link metadata
...
-				"appIDs": [
-					"com.google.test.application"
-				],
-				"components": [
-					{
-						"/": "/*",
-						"comment": "handle all urls"
-					}
-				]
+				"appID": "com.google.test.application",
```
Comment (wescarr) [inline docs/link_handling.md:51]: The `appID` is missing the Team ID prefix:
> The value of the appID key is the team ID or app ID prefix, followed by the bundle ID

You may want update the mobile app form to include a field for the Team ID, or mention that it should be included in the Application ID

**Review by wescarr — COMMENTED** [review]
Just noticed 2 potential issues. Thanks for the quick turn around

**Pull request merged**